<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>presetTrjDialog</class>
 <widget class="QDialog" name="presetTrjDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A13">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>329</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A13</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A0">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>329</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A0</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_VELTEST_CONFIG_CANCEL">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>370</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_VELTEST_CONFIG_OK">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>370</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A4">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>148</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A3">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>108</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A6">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>148</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A5">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>148</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A1">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>108</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A2">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>108</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>安全自毁曲线</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A12">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>268</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A12</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A7">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>208</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A7</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A11">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>268</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A11</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A8">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>208</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A8</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A10">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>238</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A10</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_TRJTYPE_A9">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>238</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A9</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>179</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>72</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>71</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>177</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>极性曲线</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>178</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>299</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>298</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>其他曲线</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>300</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="GROUP_CH">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>通道</string>
   </property>
   <widget class="QRadioButton" name="RADIO_CHNUM_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RADIO_CHNUM_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>26</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RADIO_CHNUM_2">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>26</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RADIO_CHNUM_3">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>26</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RADIO_CHNUM_4">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>26</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
